ProffieOS.ino

```diff
@@ -65,7 +65,7 @@
 #define SAVE_VOLUME
 #define SAVE_PRESET
 #define SAVE_COLOR_CHANGE
-#define SAVE_DYNAMIC_DIMMING
```

Comment:
This change does not belong in this PR

Reply:
Not sure why it appears as a change in the PR, it's 'SAVE BLADE DIMMING' both in the master branch and our commit.